docs/integrations/sources/surveymonkey.md

```diff
@@ -1,53 +1,53 @@
-# SurveyMonkey
+# Set up the SurveyMonkey Source Connector
 
-This page guides you through the process of setting up the SurveyMonkey source connector.
-
-:::note
-
-OAuth for Survey Monkey is officially supported only for the US. We are testing how to enable it in the EU at the moment. If you run into any issues, please [reach out to us](mailto:product@airbyte.io) so we can promptly assist you.
-
-:::
 
 <!-- env:oss -->
 
 ## Prerequisites
 
+**SurveyMonkey:**
+- Registered Private application [here](https://developer.surveymonkey.com/apps/)
+- SurveyMonkey authorization credentials
+- SurveyMonkey authorization method (oauth2.0)
+
 **For Airbyte Open Source:**
 
 - Access Token
+- Start date for the Source Connector (in UTC)
```

Comment:
I looked at the setup fields in both Open Source and Cloud. Start date is required on any of the platforms. The only distinction for Open Source is that we need an Access Token. I'll suggest one was we can make this clearer.